snuba/clickhouse/query_dsl/accessors.py

```diff
@@ -122,7 +124,10 @@ def get_time_range_expressions(
         if match is not None:
             timestamp = cast(datetime, match.scalar("timestamp"))
             assert isinstance(c, FunctionCallExpr)
-            if match.string("operator") == OPERATOR_TO_FUNCTION[">="]:
+            if match.string("operator") in (
+                OPERATOR_TO_FUNCTION[">="],
+                OPERATOR_TO_FUNCTION[">"],
```

Comment:
Is time condition always provided as a range with SnQL? ie. exact timestamp with only "=" isn't supported

Reply:
I fixed this.